lib/handlers.js

```diff
@@ -72,7 +76,7 @@ module.exports = function (dataset, opts) {
       guid: opts.guid,
       method: 'GET',
       params: params,
-      path: '/' + dataset
+      path: pathMod.normalize('/' + opts.pathToPrepend + '/' + dataset)
```

Comment:
I recently learned the _path_ module will change forward slash to backslash when used on windows. Unlikely to be an issue for us since we know 99% of the time people use Ubuntu or Mac. We could instead use something like this to safely join the url path segments: https://www.npmjs.com/package/url-join

I've not tested it, but looks like it could work.